config/locales/provider_interface/make_offer.yml

```diff
@@ -43,3 +56,5 @@ en:
           attributes:
             decision:
               blank: Select if you want to make an offer or reject the application
+            course_option_id:
```

Comment:
can we add a custom error for the course selection as well?

Reply:
added =)